tests/ignite/handlers/test_lr_finder.py

```diff
@@ -275,6 +275,18 @@ def test_lr_policy(lr_finder, to_save, dummy_engine, dataloader):
     assert all([lr[i - 1] < lr[i] for i in range(1, len(lr))])
 
 
+@pytest.mark.parametrize("step_mode", ["exp", "linear"])
+def test_multi_opt(lr_finder, dummy_engine_mulitple_param_groups, to_save_mulitple_param_groups, dataloader, step_mode):
+    start_lr = [0.1, 0.1, 0.01]
+    end_lr = [1.0, 1.0, 1.0]
+    dummy_engine = dummy_engine_mulitple_param_groups
```

Comment:
I forgot to tell you calling `./tests/run_code_style.sh fmt` applies formatting rules automatically and there were no need to redefine variables with short names. But no problem , I approve the changes. Applying that command would result in:

```python
def test_multiple_optimizers(
    lr_finder, dummy_engine_mulitple_param_groups, to_save_mulitple_param_groups, dataloader, step_mode
):
    start_lr = [0.1, 0.1, 0.01]
    end_lr = [1.0, 1.0, 1.0]

    with lr_finder.attach(
        dummy_engine_mulitple_param_groups,
        to_save_mulitple_param_groups,
        start_lr=start_lr,
        end_lr=end_lr,
        step_mode=step_mode,
    ) as trainer:
        trainer.run(dataloader)
```

Reply:
For more information you could read this [PEP guide](https://peps.python.org/pep-0008/#code-lay-out). Thank you @Jacob208M  for your time & effort.